exampleCourse/questions/fibonacciExternal/tests/autograder.py

```diff
@@ -1,80 +1,80 @@
 import py_compile
 
 def main():
-    def test1():
+    def test1(out):
         trueAns = 1
-        studAns = fib.fib(1);
+        studAns = fib.fib(1)
```

Comment:
Yea, that's the plan for the final examples - currently just trying to get it roughly working